guides/assets/stylesrc/_main.scss

```diff
@@ -795,4 +806,40 @@ body.guide {
     }
   }
   
-} // body.guide
\ No newline at end of file
+} // body.guide
+
+html[dir="rtl"] {
+  body.guide {
+    direction: rtl;
+
+    p, table th, .wrapper,
+    #guides .guides-section-container .guides-section {
+      text-align: right;
+    }
+  } // body.guide
+
+  &:lang("ar"), &:lang("fa") {
+    body.guide {
+      h1, h2, h3, h4, h5, h6, p, dl, dd, dt, ul, ol, li,
+      header#page_header nav#feature_nav .header-logo a {
+        font-family: "Noto Sans Arabic", sans-serif !important;
+        font-optical-sizing: auto;
+        font-weight: 400;
+        font-style: normal;
+        font-variation-settings: "wdth" 100;
+      }
+    } // body.guide
+  }&:lang("ar"), &:lang("fa")
+
+  &:lang("he"), &:lang("iw") {
+    body.guide {
+      h1, h2, h3, h4, h5, h6, p, dl, dd, dt, ul, ol, li,
+      header#page_header nav#feature_nav .header-logo a {
+        font-family: "Heebo", sans-serif !important;
+        font-optical-sizing: auto;
+        font-weight: 400;
+        font-style: normal;
+      }
+    } // body.guide
+  } // &:lang("he"), &:lang("iw")
```

Comment:
~~It appears `:lang` accepts a list of arguments, so we can consolidate some of these calls:~~ never mind, this does not appear to be well supported.

I am however not able to see the fonts applied when I swap languages, it's using some browser/OS default. It looks like if I unquote the args to `:lang` it works. This is on Chrome. Are you seeing the same behavior?

Reply:
I see the same behavior. I've changed them to unquoted and it works. 4e66d613